packages/davinci-client/src/lib/collector.utils.ts

```diff
@@ -180,13 +180,22 @@ export function returnSingleValueCollector<
         value: '',
         type: field.type,
       },
-      output: {
-        key: field.key,
-        label: field.label,
-        type: field.type,
-        value: data || '',
-        options: options,
-      },
+      output:
+        field.key !== undefined
```

Comment:
I don't believe we should do this. If there's no `key` property, there's something fundamentally wrong with the data. If this is a `SingleValueCollector`, it _requires_ a key in order to send the value back to the server. Without a key, we can't assign the collected value to something and the whole flow will break.